src/components/help/help.test.js

```diff
@@ -0,0 +1,211 @@
+        CypressMountWithProviders(<HelpComponent as="foo" />);
+        cy.wrap(errors).then(() => {
+          if (errors.getCalls().length) {
+            const consoleError = errors.getCalls()[0].args[0];
+            const consoleErrorTrim = consoleError.replace("\n", "");
+            expect(consoleErrorTrim).to.equal(error);
+          }
+        });
+      });
+    });
+
+    it.each(["bottom", "left", "right", "top"])(
+      "should check %s position of tooltip for Help component",
+      (position) => {
+        CypressMountWithProviders(
+          <HelpComponent tooltipPosition={position} isFocused>
+            {`This tooltip is positioned ${position}`}
+          </HelpComponent>
+        );
+        helpPreview().should("be.visible").and("have.css", position);
+      }
+    );
+
+    it.each([
+      ["carbon", "https://carbon.sage.com"],
+      ["google", "https://www.google.com"],
+    ])("should check %s href for Help component", (name, link) => {
+      CypressMountWithProviders(<HelpComponent href={link} />);
+      helpPreview().should("have.attr", "href", link);
+    });
+
+    it.each([true, false])(
+      "should check when isFocused is %s for Help component",
+      (boolVal) => {
+        CypressMountWithProviders(
+          <HelpComponent isFocused={boolVal}>{tooltipText}</HelpComponent>
+        );
+        if (boolVal === true) {
+          getDataElementByValue("tooltip").should("be.visible");
+        } else {
+          getDataElementByValue("tooltip").should("not.exist");
+        }
+      }
+    );
+
+    it.each([
+      ["orange", "rgb(255, 156, 75)"],
+      ["red", "rgb(205, 56, 75)"],
+      ["black", "rgb(0, 0, 0)"],
+      ["brown", "rgb(105, 61, 57)"],
+    ])(
+      "should check tooltip background-color as %s for Help component",
+      (name, color) => {
+        CypressMountWithProviders(
+          <HelpComponent tooltipBgColor={color} isFocused>
+            {tooltipText}
+          </HelpComponent>
+        );
+        getDataElementByValue("tooltip").should(
+          "have.css",
+          "background-color",
+          color
+        );
+      }
+    );
+
+    it.each([
+      ["orange", "rgb(255, 156, 75)"],
+      ["red", "rgb(205, 56, 75)"],
+      ["black", "rgb(0, 0, 0)"],
+      ["brown", "rgb(105, 61, 57)"],
+    ])(
+      "should check tooltip font color as %s for Help component",
+      (name, color) => {
+        CypressMountWithProviders(
+          <HelpComponent tooltipFontColor={color} isFocused>
+            {tooltipText}
+          </HelpComponent>
+        );
+        getDataElementByValue("tooltip").should("have.css", "color", color);
+      }
+    );
+
+    it.each(["top", "bottom", "right", "left"])(
+      "should check flip position to the %s for Help component",
+      (position) => {
+        CypressMountWithProviders(
+          <HelpComponent isFocused tooltipFlipOverrides={[position]}>
```

Comment:
Why do we need pass `position` as an array here ?
```suggestion
          <HelpComponent isFocused tooltipFlipOverrides={position}>
```